handbook/company/pricing-features-table.yml

```diff
@@ -649,7 +653,8 @@
   pricingTableCategories: [Device management]
   waysToUse:
     - description: Send software vulnerability emails to end users to encourage self-remediation.
-- industryName: User-initiated enrollment of macOS computers
+- industryName: Automated enrollment
```

Comment:
@noahtalerman, I missed this one. I renamed and added a tooltip. “User-initiated enrollment” is a deprecated Jamf feature that we shouldn’t use in our language. We could add “Account-driven user enrollment” (which I think "User-initiated..." is referring to) as a separate line item, but I don’t think it’s needed at this point. It might come up with iOS, though. 

cc @nonpunctual

Reply:
Hey @mike-j-thomas I think what we mean by "User-initiated enrollment" is what folks think of as "BYOD enrollment" or "Manual enrollment."

"Automated enrollment" is synonymous with "Zero-touch enrollment" 

What Jamf calls "User-initiated enrollment (deprecated)" is enrolling a specific user on a macOS host (instead of the host itself). Apple has released "Account-driven enrollment" to handle this workflow. Fleet today supports neither because it's not a common use case in the enterprise.

Reply:
I committed the change from "Automated enrollment" to "BYOD enrollment" and approved the PR, so I'm not blocking the merge if the changes look good to you.